### ISSUE TITLE
Histograms

### ISSUE THREAD

**Issue by idontgetoutmuch** [cchalmers/plots#3]
It's important not to confuse a histogram with its graphical rendition; histograms have an existence independent of how they are rendered. Too many graphics libraries conflate the two. I recommend using https://hackage.haskell.org/package/histogram-fill to manipulate histograms. A good plotting package should just concern itself with how to render such structures.

**Comment by cchalmers** [comment]
You're right, there should certainly be a way to accept already computed histogram data. But I still think there should be helper functions for plotting a histogram from sample data. `histogram-fill` looks pretty comprehensive but it's also somewhat complicated and not particularly well documented (just now, it took me a good 5-10 mins just to work out to calculate a simple histogram and get the data from it).
